testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestAvroSchemaEvolution.java

```diff
@@ -85,72 +85,72 @@ public void testInsertAfterSchemaEvolution()
                         row("int_col", "integer", "", ""));
-        assertThat(query(SELECT_STAR))
+        assertThat(onTrino().executeQuery(SELECT_STAR))
                 .containsExactlyInOrder(row("string0", 0));
 
         alterTableSchemaTo(CHANGE_COLUMN_TYPE_SCHEMA);
-        assertThat(query(COLUMNS_IN_TABLE))
+        assertThat(onTrino().executeQuery(COLUMNS_IN_TABLE))
                 .containsExactlyInOrder(
                         row("string_col", "varchar", "", ""),
                         row("int_col", "bigint", "", ""));
-        assertThat(query(SELECT_STAR))
+        assertThat(onTrino().executeQuery(SELECT_STAR))
                 .containsExactlyInOrder(row("string0", 0));
 
         alterTableSchemaTo(ADDED_COLUMN_SCHEMA);
-        assertThat(query(COLUMNS_IN_TABLE))
+        assertThat(onTrino().executeQuery(COLUMNS_IN_TABLE))
                 .containsExactlyInOrder(
                         row("string_col", "varchar", "", ""),
                         row("int_col", "integer", "", ""),
                         row("int_col_added", "integer", "", ""));
-        assertThat(query(SELECT_STAR))
+        assertThat(onTrino().executeQuery(SELECT_STAR))
                 .containsExactlyInOrder(row("string0", 0, 100));
 
         alterTableSchemaTo(REMOVED_COLUMN_SCHEMA);
-        assertThat(query(COLUMNS_IN_TABLE))
+        assertThat(onTrino().executeQuery(COLUMNS_IN_TABLE))
                 .containsExactlyInOrder(row("int_col", "integer", "", ""));
-        assertThat(query(SELECT_STAR))
+        assertThat(onTrino().executeQuery(SELECT_STAR))
                 .containsExactlyInOrder(row(0));
 
         alterTableSchemaTo(RENAMED_COLUMN_SCHEMA);
-        assertThat(query(COLUMNS_IN_TABLE))
+        assertThat(onTrino().executeQuery(COLUMNS_IN_TABLE))
                 .containsExactlyInOrder(
                         row("string_col", "varchar", "", ""),
                         row("int_col_renamed", "integer", "", ""));
-        assertThat(query(SELECT_STAR))
+        assertThat(onTrino().executeQuery(SELECT_STAR))
                 .containsExactlyInOrder(row("string0", null));
     }
 
     @Test(groups = AVRO)
     public void testSchemaWhenUrlIsUnset()
     {
-        assertThat(query(COLUMNS_IN_TABLE))
+        assertThat(onTrino().executeQuery(COLUMNS_IN_TABLE))
                 .containsExactlyInOrder(
                         row("string_col", "varchar", "", ""),
                         row("int_col", "integer", "", ""));
-        assertThat(query(SELECT_STAR))
+        assertThat(onTrino().executeQuery(SELECT_STAR))
                 .containsExactlyInOrder(row("string0", 0));
 
-        onHive().executeQuery(format("ALTER TABLE %s UNSET TBLPROPERTIES('avro.schema.url')", TABLE_NAME));
-        assertThat(query(COLUMNS_IN_TABLE))
+        onHive().executeonTrino().executeQuery(format("ALTER TABLE %s UNSET TBLPROPERTIES('avro.schema.url')", TABLE_NAME));
```

Comment:
:)

Reply:
Yep noticed it all-ready :) not a case anymore :)